### PR TITLE
[READY] Misc Fulpstation Quality of Life Changes

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request
In this PR, I brought back:

- The Fulpstation title screen
- The Fulpstation lobby musics
- The Fulpstation greeting message on the arrival shuttle (would've been weird otherwise)
- The Fulpstation credits music (when the shift ends) [Now fully modular and easy to edit/add more]
- The custom list of round end sounds (when the server restarts, wasn't exactly the same as /tg/'s and I also brought back one they removed or that we had as an extra) [Now fully modular and easy to edit/add more]
- The space-walking ambient music that Mokoshotar made [Now fully modular and easy to edit/add more]
- The old MOTD (~~links are a bit broken right now, if I don't find a fix I'll just change it again to point to the top of the screen, which should contain working links~~ Links were fixed thanks to my bug report, it should now work when we update to more recent /tg/)
- The randomized station names (although that's in the configs and I'm not sure if I'll keep that in this PR, might be something that Wind wants to look at doing himself directly)

I made sure that all the code that could easily be made modular, would be made modular. Since the title screen and just title stuff in general is in the .gitignore, I'll be making a PR after this one goes through to just fix the .gitignore if it messes with the repo's. Title screen and lobby music were done the same way as they were on the original repo, mostly because the code behind them is a bit more complex and I didn't see an easy way to make it more modular than it is, and to be quite honest with you, /tg/ won't be modifying those anyway, so it doesn't truly matter.

Basically, I made it as easy as I could to add some new ambience music, credits music and round-end sounds, with minimal touching of the /tg/ files to avoid future merge conflicts.

~I haven't added anything new yet, I'll probably do that in a future PR so there's no conflict of interests, as this is basically just a port of what we had before, just written in a less intrusive way on /tg/ files.~
I just added one round-end track, the classic and highly-requested "Anyways, I got banned on Fulpstation".

## Why It's Good For The Game

Bringing back Fulp-only visuals and sounds is just a must, it's what sets us apart from the other stations at first glance. Obviously, I might've missed some stuff, but that's why I'm making this PR now, so I can get some more feedback from people.

## Changelog
:cl:
soundadd: Added back the Fulp lobby musics.
soundadd: Added back the Fulp arrival shuttle greeting message.
soundadd: Added back the Fulp credits musics.
soundadd: Added back the Fulp round restart sounds.
imageadd: Added back the Fulp lobby screen.
code: More modularity to some sound code to make it easy to change without touching /tg/ files.
refactor: refactored some code
config: Re-enabled random station names
config: Added back the MOTD
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
Also, for those who were wondering, the linter error is due to something that was fixed on the upstream, just a folder named strangely (they added a .dm extension to the folder for one reason or another).
https://github.com/tgstation/tgstation/pull/56193
^ 
Here's the PR that fixed it, just in case you're actually curious

File: .gitignore [hunk 1]
```diff
@@ -218,10 +218,10 @@ tools/CreditsTool/obj/*
...
 /config/jukebox_music/sounds/*
 !/config/jukebox_music/sounds/exclude
-/config/title_music/sounds/*
-!/config/title_music/sounds/exclude
-/config/title_screens/images/*
-!/config/title_screens/images/exclude
+# /config/title_music/sounds/*
+# !/config/title_music/sounds/exclude
+# /config/title_screens/images/*
+# !/config/title_screens/images/exclude
```
[the remark this answers is not in the document]
Comment (GoldenAlpharex) [reply .gitignore:224]: Hmmm, I haven't really found where that'd be. What I'm doing is just changing the .gitignore temporarily so I can add these via this PR, I'll be reverting the .gitignore once this PR goes through.

**Comment by JohnFulpWillard** [comment]
Why was ApproachingTG.ogg deleted?

**Comment by Fikou** [comment]
maybe change the config so it links to fulp wiki/github instead of tg

**Comment by remove32** [comment]
> 
> 
> maybe change the config so it links to fulp wiki/github instead of tg

i made that change in the local config but never committed to github because i'm braindead

**Comment by GoldenAlpharex** [comment]
> Why was ApproachingTG.ogg deleted?

It wasn't, it was switched with another file instead, the one we're using on Fulp.

File: code/__HELPERS/roundend.dm [hunk 2]
```diff
@@ -366,8 +366,8 @@
  *
  * Composits the roundend report, and saves it in two locations.
  * The report is first saved along with the round's logs
- * Then, the report is copied to a fixed directory specifically for
```
Comment (JohnFulpWillard) [inline code/__HELPERS/roundend.dm:369]: lol why edit tg comments of all things

File: code/controllers/subsystem/ticker.dm [hunk 3]
```diff
@@ -662,17 +662,23 @@ SUBSYSTEM_DEF(ticker)
...
-		'sound/roundend/disappointed.ogg',
-		'sound/roundend/scrunglartiy.ogg',
-		'sound/roundend/petersondisappointed.ogg'\
-		)
+		// FULP-ONLY STARTS - Adds a pointer to the new modular pick_round_end_sound() proc from Fulp's sound.dm
+		// Original code left in comment
+
+		round_end_sound = pick_round_end_sound()
+
+		// round_end_sound = pick(\
```
Comment (JohnFulpWillard) [inline code/controllers/subsystem/ticker.dm:670]: couldnt you comment this out with /* */ instead, so TG conflicts wouldnt actually conflict as much?

File: config/config.txt [hunk 4]
```diff
@@ -22,7 +22,7 @@ $include interviews.txt
 # SERVERSQLNAME tgstation
 
 ## Station name: The name of the station as it is referred to in-game. If commented out, the game will generate a random name instead.
-STATIONNAME Space Station 13
+#STATIONNAME Space Station 13
```
Comment (JohnFulpWillard) [inline config/config.txt:25]: why are you editing config files

File: config/motd.txt [hunk 5]
```diff
@@ -1,5 +1,11 @@
 <!-- Most valid html will work in here, excluding images. -->
 
-<h1>Welcome to Space Station 13!</h1>
+<h1>Welcome to Fulpstation!</h1>
```
Comment (JohnFulpWillard) [inline config/motd.txt:3]: Again, this is config, this should be done locally by wind.

**Review by JohnFulpWillard — CHANGES_REQUESTED** [review]

File: code/controllers/subsystem/ticker.dm (hunk 3 above)
Comment (JohnFulpWillard) [reply code/controllers/subsystem/ticker.dm:670]: Also it would likely fix the linter issue you have, instead of keep blaming the master being bad.

**Comment by Helianthus00** [comment]
Address John's reviews and I can merge this.

File: config/config.txt (hunk 4 above)
Comment (GoldenAlpharex) [reply config/config.txt:25]: It was back when I asked Wind if he wanted me to do this, and it was so people would receive the changes on their end when they'd download the repo. I'll probably revert that, though.

File: config/motd.txt (hunk 5 above)
Comment (GoldenAlpharex) [reply config/motd.txt:3]: Once again, that was back when it wasn't done on the repo and I just wanted to help out Wind at that time. It doesn't really matter anymore, so I'll probably just revert that.

File: code/__HELPERS/roundend.dm (hunk 2 above)
Comment (GoldenAlpharex) [reply code/__HELPERS/roundend.dm:369]: I'll be honest, I don't know what's up, I didn't touch those comments. I might have to webedit as it's something I believe VSC did on its own and I'm unable to fix it through there.

**Comment by GoldenAlpharex** [comment]
> Address John's reviews and I can merge this.

Done.

**Pull request merged**